src/dataprotection/azext_dataprotection/manual/enums.py

```diff
@@ -65,6 +65,13 @@ def get_secret_store_type_values():
     return ['AzureKeyVault']
 
 
+def get_backup_operation_values():
+    return ['Backup']
+
+def get_permission_scope_values():
+    return ['Resource', 'Resource Group', 'Subscription']
```

Comment:
Check with Aditya Balaji once on the naming of acceptable parameters. We usually don't allow spaces between an argument like in Resource Group. As it will require users to pass the argument in quotes.